python/mach_bootstrap.py

```diff
@@ -78,14 +78,8 @@
 # Possible names of executables
 # NOTE: Windows Python doesn't provide versioned executables, so we must use
 # the plain names. On MSYS, we still use Windows Python.
-if sys.platform in ['msys', 'win32']:
-    PYTHON_NAMES = ["python"]
-    VIRTUALENV_NAMES = ["virtualenv"]
-    PIP_NAMES = ["pip"]
-else:
-    PYTHON_NAMES = ["python-2.7", "python2.7", "python2", "python"]
-    VIRTUALENV_NAMES = ["virtualenv-2.7", "virtualenv2.7", "virtualenv2", "virtualenv"]
-    PIP_NAMES = ["pip-2.7", "pip2.7", "pip2", "pip"]
+PYTHON_NAMES = ["python-2.7", "python2.7", "python2", "python"]
+VIRTUALENV_NAMES = ["virtualenv-2.7", "virtualenv2.7", "virtualenv2", "virtualenv"]
```

Comment:
These names aren't used now.

Reply:
They're still used for finding the binaries on the inside of the virtualenv.

Reply:
I mean VIRTUALENV_NAMES isn't used now.